src/color/rgbToHex.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Convert the given RGB color value to hex color value (string, used in CSS, etc)
+ * Conversion forumla:
+ * - convert R, G, B into HEX strings
+ * - return HEX formatted string "#RRGGBB"
+ * @param  {Float} r - red value
+ * @param  {Float} g - green value
+ * @param  {Float} b - blue value
+ */
+const rgbToHex = (r, g, b) => {
+  if (Array.isArray(r)) {
+    b = r[2]
+    g = r[1]
+    r = r[0]
+  }
+  let s = '#' +
```

Comment:
es6 strings please :)

Reply:
I saw it... but the conversation to string is a wild one.